Docs/Configuration.tex

```diff
@@ -2766,126 +2775,135 @@ \section{Misc}\label{misc}
 \end{enumerate}
 
-The display order of the boot options in the picker and the boot process are determined separately from the scanning
-algorithm. The display order as follows:
+The display order of the boot options in the boot picker and the boot process are determined separately from the scanning
+algorithm.
+
+The display order as follows:
 
 \begin{itemize}
 \tightlist
-\item Alternate options follow corresponding primary options, i.e. Apple recovery will be following the
+\item Alternate options follow corresponding primary options. That is, Apple recovery options will follow the
   relevant macOS option whenever possible.
-\item Options will be listed in file system handle firmware order to maintain an established order across
-  the reboots regardless of the chosen operating system for loading.
+\item Options will be listed file system handle firmware order to maintain an established order across
+  reboots regardless of the operating system chosen for loading.
 \item Custom entries, tools, and system entries will be added after all other options.
-\item Auxiliary options will only show upon entering ``Advanced Mode'' in the picker (usually by pressing ``Space'').
+\item Auxiliary options will only be displayed upon entering ``Extended Mode'' in the boot picker (typically by pressing the \texttt{Space} key).
 \end{itemize}
 
 The boot process is as follows:
 \begin{itemize}
 \tightlist
-\item Try looking up first valid primary option through \texttt{BootNext} UEFI variable.
-\item On failure looking up first valid primary option through \texttt{BootOrder} UEFI variable.
+\item Look up the first valid primary option in the \texttt{BootNext} UEFI variable.
+\item On failure, look up the first valid primary option in the \texttt{BootOrder} UEFI variable.
 \item Mark the option as the default option to boot.
 \item Boot option through the picker or without it depending on the \texttt{ShowPicker} option.
 \item Show picker on failure otherwise.
 \end{itemize}
 
-\emph{Note 1}: This process is meant to work reliably only when \texttt{RequestBootVarRouting}
+\emph{Note 1}: This process will only work reliably when the \texttt{RequestBootVarRouting}
 option is enabled or the firmware does not control UEFI boot options (\texttt{OpenDuetPkg} or
-custom BDS). Without \texttt{LauncherOption} it also is possible that other operating systems
-overwrite OpenCore, make sure to enable it when planning to use them.
+custom BDS). When \texttt{LauncherOption} is not enabled, other operating systems can
+overwrite OpenCore settings and this property should therefore be enabled when planning
+to use other operating systems.
 
-\emph{Note 2}: UEFI variable boot options' boot arguments will be removed if present as they
-may contain arguments compromising the operating system, which is undesired once secure boot
-is enabled.
+\emph{Note 2}: UEFI variable ``boot options'' boot arguments will be removed, if present, as
```

Comment:
This was not a quote. UEFI boot option variables are structures, which contain boot options field that is used as boot arguments for most operating systems. Please revert..